server/trillian_log_server/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
 	"github.com/google/trillian/monitoring"
-	"github.com/google/trillian/quota"
+	mysqlq "github.com/google/trillian/quota/mysql"
```

Comment:
This may be a bad idea and might not help but could we move the mysql query of the quota manager out to storage/mysql? or storage/mysql/quota?